argus/backend/controller/api.py

```diff
@@ -389,9 +389,10 @@ def release_stats_v2():
     release = request.args.get("release")
     limited = bool(int(request.args.get("limited")))
     version = request.args.get("productVersion", None)
+    include_no_version = bool(int(request.args.get("includeNoVersion", False)))
```

Comment:
I think that the default should be "True".
The more conservative approach is the default, if one wants to filter it out, he can later.

Reply:
Done